### PR TITLE
Replace classic division with floor division where applicable

### PR DESCRIPTION
**Pull request by gherceg**
Step 3 of [this document](https://docs.google.com/document/d/17QQu5BAAwle5KHTMawTlsf4o78ZRXElIKlrBdHjrU8M/edit#).

### CODE REVIEW

File: scripts/es_snapshot.py [hunk 1]
```diff
@@ -348,8 +349,8 @@ def run(self):
                 suffix = f" ({total - success} of {total} shards failed)"
             print(f"    {snapshot} [{state}]{suffix}")
             if self.detail:
-                start = datetime.fromtimestamp(snap_info['start_time']/1000)
-                end = datetime.fromtimestamp(snap_info['end_time']/1000)
+                start = datetime.fromtimestamp(snap_info['start_time']//1000)
+                end = datetime.fromtimestamp(snap_info['end_time']//1000)
```
Comment (millerdev) [inline scripts/es_snapshot.py:353]: Do we know for sure that `snap_info['start_time']` and `snap_info['end_time']` are always ints (not floats)?

**Review by millerdev — APPROVED** [review]
Approved unless the answer to my question is negative.

File: scripts/es_snapshot.py (hunk 1 above)
Comment (gherceg) [reply scripts/es_snapshot.py:353]: I think you are right and it is better to use float division here. @snopoke can you confirm?

Comment (snopoke) [reply scripts/es_snapshot.py:353]: I don't know, I'd have to check.

This script was developed for PY3 only so it shouldn't need any change.

Comment (gherceg) [reply scripts/es_snapshot.py:353]: Good point. I'll revert changes made to this file.

**Review by millerdev — APPROVED** [review]

**Pull request merged**